Monika After Story/game/script-story-events.rpy

```diff
@@ -2457,6 +2457,15 @@ label mas_notification_windowreact:
     m 3eua "I've been practicing coding a bit more and I've learned how to use the notifications on your computer!"
     m "So if you want, I can let you know if I have something for us to talk about."
 
+    if not store.mas_windowreacts.can_show_notifs:
```

Comment:
One thing. We never get to this point in the dialogue if we can't show notifs. Will have to make some more changes so we get to this point in normal flow, but adjust the opening dialogue to be more reflective that it's more research instead of practicing.

Reply:
Alright, is it better with the rewritten dialogue? As for the fact it doesn't show up, is it better to just remove the checks?

Reply:
I'll go thru this later and adjust the main topic to actually flow into this path and correct a bit of existing dlg, so it'll lead in better. But will do this later.